apps/simple-trading-app-e2e/src/integration/market-list.test.ts

```diff
@@ -25,44 +31,49 @@ describe('market list', () => {
             asset = children[1].innerText;
             if (asset) {
               cy.wrap(children[1]).click();
-              cy.location('pathname').should('equal', `/markets/all/${asset}`);
+              cy.location('pathname').should(
+                'match',
+                new RegExp(`/markets/all/Future/${asset}`, 'i')
+              );
```

Comment:
nitpick: `cy.url().should('include', '/users/1/edit')`

Reply:
here `match i` is because we need check it case insensitive